advocacy_docs/dev-guides/deploy/windows.mdx

```diff
@@ -0,0 +1,149 @@
+PostgreSQL 16 PostgreSQL.PostgreSQL.16 16.4-1  winget
+PostgreSQL 9  PostgreSQL.PostgreSQL.9  9       winget
+```
+
+Installers for release candidate versions of PostgreSQL are also available on EDB's website at https://www.enterprisedb.com/downloads/postgres-postgresql-downloads
+!!!
+
+!!! SeeAlso "Further reading"
+For more fine-grained control, you can also download and run the installer interactively by following the instructions in [Installing PostgreSQL on Windows](https://www.enterprisedb.com/docs/supported-open-source/postgresql/installing/windows/). 
+!!!
+
+## Add PostgreSQL commands to your path
+
+PostgreSQL comes with [several useful command-line tools](https://www.postgresql.org/docs/current/reference-client.html) for working with your databases. For convenience, you'll probably want to add their location to your path.
+
+1.  Open the System Properties control panel and select the Advanced tab (or run `SystemPropertiesAdvanced.exe`)
+2.  Activate the "Environment Variables..." button to open the envornment variables editor
+3.  Find the `Path` variable under the "System variables" heading, and click "Edit..."
+4.  Add the path to the bin directory of the PostgreSQL version you installed, e.g.  `C:\Program Files\postgresql\16\bin\` (where *16* is the version of PostgreSQL that you installed).
+
+!!! Warning "This only affects new command prompts"
+If you have a command prompt open already, you'll need to close and reopen in order for your changes to the system path to take effect.
+!!!
+
+## Verifying your installation
+
+If the steps above completed successfully, you'll now have a PostgreSQL service running with the default database and superuser account. Let's verify this by connecting, creating a new user and database, and then connecting using that user.
+
+### Connect with psql
+
+Open a new command prompt, and run
+
+```
+psql -U postgres
+___OUTPUT__
+Password for user postgres:
+
+psql (16.4)
+WARNING: Console code page (437) differs from Windows code page (1252)
+         8-bit characters might not work correctly. See psql reference
+         page "Notes for Windows users" for details.
+Type "help" for help.
+
+postgres=#
+```
+
+When prompted, enter the default password (`postgres`). 
+
+!!! Warning
+We're setting up an environment for local development, and have no intention of enabling remote connections or working with sensitive data - so leaving the default password is fine. On a production or multi-user system, or when working with sensitive data, you should *at minimum* set a strong password for the superuser - and realistically, should probably disable password authentication for it entirely.
+!!!
+
+It's a good practice to develop with a user and database other than the default - this helps avoid issues later on when you go to deploy and have no idea what permissions your app needs! So let's start off right by creating an app user and giving it its own database:
```

Comment:
As someone learning the space myself, I would love another sentence here expounding on why it's a good practice to develop with a user and database other than the default. How does it help avoid issues later on when you go to deploy?

Reply:
Expanded this a bit, added a link for further reading inline, and more resources at the end of the section.